scripts/run-integration-tests.sh

```diff
@@ -168,6 +169,13 @@ GO111MODULE=on go test -v -timeout 0 ./... --kubeconfig=$KUBECONFIG --ginkgo.foc
 TEST_PASS=$?
 popd
 
+if [[ $TEST_PASS -eq 0 && "$RUN_CONFORMANCE" == true ]]; then
+  echo "Running conformance tests against cluster"
+  wget -qO- https://dl.k8s.io/v$K8S_VERSION/kubernetes-test.tar.gz | tar -zxvf - --strip-components=4 -C /tmp  kubernetes/platforms/linux/amd64/e2e.test
```

Comment:
Just a quick note. We will soon [support multi-arch builds](https://github.com/aws/amazon-vpc-cni-k8s/pull/837), so eventually we may want to enable arch-specific e2e testing. Nothing to act upon here, just making a note of this.

Reply:
Thanks for the heads up.